src/models/OuterControl.jl

```diff
@@ -41,8 +41,38 @@ function OuterControl(
     )
 end
 
-function OuterControl(; active_power, reactive_power, ext = Dict{String, Any}())
-    OuterControl(active_power, reactive_power, ext)
+function OuterControl(;
+    active_power,
+    reactive_power,
+    ext = Dict{String, Any}(),
+    states = nothing,
+    n_states = nothing,
+)
+    if isnothing(states)
```

Comment:
helps inference  https://github.com/JuliaLang/julia/pull/36444
```suggestion
    if states === nothing
```
same applies to multiple places in the PR :)

Reply:
I have no problem changing to this.  However, if you're not supposed to use a Julia standard library call then maybe it shouldn't exist?  Or come with a recommendation on when to use it?